builtin/credential/ldap/path_config.go

```diff
@@ -100,6 +106,10 @@ func (b *backend) pathConfigWrite(
 	if groupdn != "" {
 		cfg.GroupDN = groupdn
 	}
+	sslverify := d.Get("sslverify").(bool)
+	if !sslverify {
+		cfg.SSLVerify = sslverify
```

Comment:
I'm a big fan of adding a very large warning message here explaining that disabling SSL verification is bad and is not secure.